store/src/main/java/org/apache/rocketmq/store/ConsumeQueue.java

```diff
@@ -330,6 +336,17 @@ public boolean flush(final int flushLeastPages) {
         return result;
     }
 
+    public int deleteExpiredFile(
```

Comment:
Does the expired topic only delete the consumeQueue, and the commitLog still exists?